azerothcore/azerothcore-wotlk
# fix(DB/waypoint_data): add waypoint_data, update creature and creature_addon

ported path from TC

## Changes Proposed:
-  add waypoint_data
-  updated creature and creature_addon

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6247
- Closes https://github.com/chromiecraft/chromiecraft/issues/793

## SOURCE:
ported from TC

## Tests Performed:
- tested in-game

## How to Test the Changes:
- `.go cr 41758`


You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
